Library/Homebrew/extend/pathname.rb

```diff
@@ -131,6 +131,12 @@ def write(content, *open_args)
     open("w", *open_args) { |f| f.write(content) }
   end
 
+  # Only appends to a file that is already created.
+  def write_append(content, *open_args)
+    raise "Cannot append file that doesn't exist: #{self}" unless exist?
+    open("a", *open_args) { |f| f.puts(content) }
```

Comment:
While I can understand the reason for using `f.puts`, I would also prefer to see it replaced with `f.write`. I think making the interface in its interpretation of the `content` argument consistent with `Pathname#write` (implemented directly above) trumps the minor inconvenience of having to add a few `\n` in some places.